performance/SwiftUIPerformanceQuickstart/PerformanceExample/Podfile

```diff
@@ -0,0 +1,34 @@
+# Uncomment the next line to define a global platform for your project
+
+target 'PerformanceExample (iOS)' do
+  platform :ios, '14.0'
```

Comment:
Now that macOS is gone, can the contents of each target be factored outside to the top as one copy instead of three?

Reply:
Done ✅